smap/sources/conf/safar_conf.ini

```diff
@@ -5,7 +5,7 @@ Port = 8003
 uuid = 6fa21782-27b2-11e2-9a49-370ecdac5e02
 
 [report 0]
-ReportDeliveryLocation = http://localhost:8079/add/MYAPIKEY
+ReportDeliveryLocation = http://localhost/raqmn_api/add/6fa21782-27b2-11e2-9a49-370ecdac5e02/
```

Comment:
Right now, we don't have any checks and this key is an example of how keys look like, this is same as uuid but as we proceed every sensor will have to register and get a unique uuid for itself which will also be used as key. This is just for an example. Actual keys wont be released.